src/edu/ithaca/dragonlab/ckc/io/CSVReader.java

```diff
@@ -103,14 +71,120 @@ else if (learningObjectList.size()+2 > singleList.size()){
+                lineList.add(lineToList(line));
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return lineList;
     }
 
-    public ArrayList<LearningObject> getManualGradedLearningObjects(){
-        return this.learningObjectList;
+
+    /**
+     * returns a list of LearningObjects from a string created from a csv file
+     * @param singleList one line of text from a csv file
+     * @return LearningObject list
+     */
+    public static List<LearningObject> loLister(ArrayList<String> singleList) {
+        int i = 2;
+        List<LearningObject> loList = new ArrayList<LearningObject>();
+        while(i<singleList.size()){
+            String question = singleList.get(i);
+            //used to find the max score of a question (won't be affected if there are other brackets in the question title
+            int begin = question.lastIndexOf('[');
+            int end = question.lastIndexOf(']');
+
+
+            //TODO: temp check to see what problem is... not a proper solution to bug #76
+            if (begin >= 0 && end >= 0) {
+                String maxScoreStr = question.substring(begin + 1, end);
+                double maxScore = Double.parseDouble(maxScoreStr);
+                question = question.substring(0, begin - 1);
+                LearningObject learningObject = new LearningObject(question);
+                learningObject.setMaxPossibleKnowledgeEstimate(maxScore);
+                loList.add(learningObject);
+            }
+            else {
+                //logger.error("No max score found for string:"+question+"\t defaulting to 1, which is probably wrong");
+                LearningObject learningObject = new LearningObject(question);
+                learningObject.setMaxPossibleKnowledgeEstimate(1);
+                loList.add(learningObject);
+            }
+            i++;
+        }
+        return loList;
+    }
+
+
+    /**
+     * takes a list of csv files and creates a single list of LearningObjects from all files
+     * @param csvfiles
+     * @return a list of all LearningObjects across all files
+     */
+    public static List<LearningObject> fullLoLister(List<String> csvfiles){
+        List<LearningObject> fullLoList = new ArrayList<LearningObject>();
+
+        //Each csvfile has their LOs searched
+        for(String file: csvfiles){
+            ArrayList<ArrayList<String>> lineList = CSVReader.staticLineToList(file);
+            List<LearningObject> loList = new ArrayList<LearningObject>();
+            loList = CSVReader.loLister(lineList.get(0));
+
+            //adding current csvfile's LOs to the full list of LOs
+            for(LearningObject learningObject: loList) {
+                fullLoList.add(learningObject);
+            }
+        }
+        return fullLoList;
     }
 
+    public void lorLister(ArrayList<String> singleList, int i, Logger logger)throws NullPointerException{
```

Comment:
Particularly void functions need to have javadoc comments with a post tag, to say what the function does. Also, since this is a void function that changes data members, should it be private, or do you want outside people calling this function?